docs/vector/vector-intro.md

```diff
@@ -45,28 +46,28 @@ val nonIntersection = myPoint & Point(200, 300)
 
 At this point, the values `selfIntersection` and `nonIntersection`
 are `GeometryResult` containers. These containers are what many JTS
-operations on `Geometry` objects will wrap their results in. Source
-for the To idiomatically destructure these wrappers, we do the following:
+operations on `Geometry` objects will wrap their results in.
+To idiomatically destructure these wrappers, we can use the
+`as[G <: Geometry]` function which either returns `Some(G)` or `None`.
 
 ```scala
-def unwrapPoint(res: PointGeometryIntersectionResult): Option[Point] =
-  res match {
-    case PointResult(point) => Some(point)
-    case _ => None
-  }
-// Et voila:
-assert(unwrapPoint(selfIntersection) == Some(myPoint))  // Either some point
-assert(unwrapPoint(nonIntersection) == None)  // Or nothing at all
+val pointIntersection = (Point(1.0, 2.0) & Point(1.0, 2.0)).as[Point]
+val pointNonIntersection = (Point(1.0, 2.0) & Point(12.0, 4.0)).as[Point]
+
+assert(pointIntersection == Some(Point(1.0, 2.0)))  // Either some point
+assert(pointNonIntersection == None)                // Or nothing at all
 ```
 
 Beyond the methods which come with any `Geometry` object there are
 implicits in many geotrellis modules which will extend Geometry
 capabilities. For instance, after importing `geotrellis.vector._`,
```

Comment:
I changed this to be similar to the rest of the library, where we are now placing implicits in the root package, so this is actually correct.